pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -898,6 +898,28 @@ var _ = Describe("Clone", func() {
 			})
 		})
 
+		Context("Target VM name", func() {
+			expectTargetVMNameExist := func() {
+				restore, err := client.SnapshotV1beta1().VirtualMachineRestores(metav1.NamespaceDefault).Get(context.TODO(), testRestoreName, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(restore.Spec.VirtualMachineSnapshotName).To(Equal(testSnapshotName))
+				patchedVM, err := offlinePatchVM(sourceVM, restore.Spec.Patches)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(patchedVM.Name).ToNot(BeNil())
+			}
```

Comment:
I don't think we need this function since its content can be executed directly in the `It` body.
Let's consider it as follow-up.
/lgtm